sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py

```diff
@@ -27,7 +27,7 @@ class MsalCredential(object):
     def __init__(self, client_id, client_credential=None, **kwargs):
         # type: (str, Optional[Union[str, Dict]], **Any) -> None
         authority = kwargs.pop("authority", None)
-        self._validate_authority = kwargs.pop("validate_authority", True)
+        # self._validate_authority = kwargs.pop("validate_authority", True)
```

Comment:
It might be worth including a link to the issue that motivated commenting this out, just for future reference. Also fine if we don't, since I assume the kwarg will be back in the near future

Reply:
We will do the same as last month. 

Release a stable version w/o this feature and a beta version with this feature. :)